miscellaneous/metaDataRemover/main.go

```diff
@@ -0,0 +1,185 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strconv"
+	"strings"
+
+	logger "github.com/ElrondNetwork/elrond-go-logger"
+	"github.com/ElrondNetwork/elrond-tools-go/miscellaneous/metaDataRemover/config"
+	"github.com/ElrondNetwork/elrond-tools-go/trieTools/trieToolsCommon"
+	"github.com/pelletier/go-toml"
+	"github.com/urfave/cli"
+)
+
+const (
+	logFilePrefix   = "meta-data-remover"
+	tomlFile        = "./config.toml"
+	outputFilePerms = 0644
+)
+
+func main() {
+	app := cli.NewApp()
+	app.Name = "Tokens exporter CLI app"
+	app.Usage = "This is the entry point for the tool that deletes tokens meta-data"
+	app.Flags = getFlags()
+	app.Authors = []cli.Author{
+		{
+			Name:  "The Elrond Team",
+			Email: "contact@elrond.com",
+		},
+	}
+
+	app.Action = func(c *cli.Context) error {
+		return startProcess(c)
+	}
+
+	err := app.Run(os.Args)
+	if err != nil {
+		log.Error(err.Error())
+		os.Exit(1)
+		return
+	}
+}
+
+func startProcess(c *cli.Context) error {
+	flagsConfig := getFlagsConfig(c)
+
+	_, errLogger := trieToolsCommon.AttachFileLogger(log, logFilePrefix, flagsConfig.ContextFlagsConfig)
+	if errLogger != nil {
+		return errLogger
+	}
+
+	log.Info("sanity checks...")
+
+	err := logger.SetLogLevel(flagsConfig.LogLevel)
+	if err != nil {
+		return err
+	}
+
+	log.Info("starting processing", "pid", os.Getpid())
+
+	shardTokensMap, err := readTokensInput(flagsConfig.Tokens)
+	if err != nil {
+		return err
+	}
+
+	cfg, err := loadConfig()
+	if err != nil {
+		return err
+	}
+
+	shardTxsDataMap, err := createShardTxsDataMap(shardTokensMap, cfg.TokensToDeletePerTransaction)
+	if err != nil {
+		return err
+	}
+
+	shardPemsDataMap, err := readPemsData(flagsConfig.Pems)
+	if err != nil {
+		return err
+	}
+
+	return createShardTxs(flagsConfig.Outfile, cfg, shardPemsDataMap, shardTxsDataMap)
+}
+
+func readTokensInput(tokensFile string) (map[uint32]map[string]struct{}, error) {
+	workingDir, err := os.Getwd()
+	if err != nil {
+		return nil, err
+	}
+
+	fullPath := filepath.Join(workingDir, tokensFile)
+	jsonFile, err := os.Open(fullPath)
+	if err != nil {
+		return nil, err
+	}
+
+	bytesFromJson, err := ioutil.ReadAll(jsonFile)
+	if err != nil {
+		return nil, err
+	}
+
+	shardTokensMap := make(map[uint32]map[string]struct{})
```

Comment:
as in the other PR, this is hard to follow: `map[uint32]map[string]struct{}` might represent anything in any form. Suggestion to change to a proper struct